server/services/matches.ts

```diff
@@ -62,4 +62,26 @@ const createMatchService = async (
   };
 };
 
-export default createMatchService;
+export const getAllMatches = async () => {
+  const currentDate = Date.now();
+  const currentDateObject = new Date(currentDate);
+  const currentDateFormated = currentDateObject.toISOString();
+
+  const matches = await Match.findAll({
```

Comment:
we want to git the stadium left joined whit matches 
because we must use it on the match card : 
![Screenshot from 2023-06-22 23-39-05](https://github.com/GSG-G13/Balanty/assets/100975982/eff760ee-223c-40af-b195-bbf72c55e9b6)